docs/demo/gaiden-css/components/media/default/demo.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="media col-small-12 col-medium-4 col-normal-4">
+      <div class="media__image thumbnail space-box-medium">
```

Comment:
@MarcoBrunoBR if it doesn't break the existing layout I'm ok with it. If it breaks, I would suggest to discuss semantics in another opportunity. What do you think @eduardojmatos ?